WebHostLib/static/assets/tutorial/archipelago/commands_en.md

```diff
@@ -91,5 +91,6 @@ including the exclamation point.
 - `/allow_forfeit <player name>` Allows the given player to use the `!forfeit` command.
 - `/forbid_forfeit <player name>` Bars the given player from using the `!forfeit` command.
 - `/send <player name> <item name>` Grants the given player the specified item.
+- `/yeet <amount> <player name> <item name>` Grants the given player the stated amount of the specified item.
```

Comment:
Updated docs, yesssssssss 👍👍👍👍👍

Reply:
I feel like my IDE deserves the praise, i just searched for `/send` everywhere xD